app/addons/components/react-components.react.jsx

```diff
@@ -1440,35 +1430,39 @@ function (app, FauxtonAPI, React, ReactDOM, Actions, Stores, FauxtonComponents,
     },
 
     onInputChange: function (e) {
+      var val = encodeURIComponent(e.target.value.trim());
+
       this.setState({
-        inputValue: e.target.value
+        inputValue: val
       });
 
       this.setState({
-        disableSubmit: e.target.value.trim() !== this.props.showModal.dbId.trim()
+        disableSubmit: val !== this.props.modalProps.dbId.trim()
```

Comment:
Tempting to put the `this.props.modalProps.dbId.trim()` into a `getDbId()` helper since it's done several times.